ios/RCCLightBox.m

```diff
@@ -48,9 +48,10 @@ -(instancetype)initWithFrame:(CGRect)frame params:(NSDictionary*)params
                 UIColor *backgroundColor = [RCTConvert UIColor:style[@"backgroundColor"]];
                 if (backgroundColor != nil)
                 {
-                    self.overlayColorView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, frame.size.width, frame.size.height)];
+                    self.overlayColorView = [[UIView alloc] init];
                     self.overlayColorView.backgroundColor = backgroundColor;
                     self.overlayColorView.alpha = 0;
+                    [self.overlayColorView setTranslatesAutoresizingMaskIntoConstraints:NO];
```

Comment:
Do we still need this? It seems that since we're manually setting the bounds anyway, we don't need to toggle anything with the autoresizing mask.